src/test/scala/beam/router/PhyssymXmlToOsmConverter.scala

```diff
@@ -62,6 +68,7 @@ object PhyssymXmlToOsmConverter extends StrictLogging {
               startElement.getAttributeByName(new QName("to")).getValue
             )
             wayInnerTags = buildLinkProps(startElement)
+            links += Way(wayId, source_destination._1, source_destination._2, wayInnerTags)
```

Comment:
Also, you added to the links here, but removed from the endElement section. Shouldn't it still be added there also?

Reply:
nice catch. this change should not have been done